# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_bin86.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bin86-0.16.12/ld/Makefile.orig	Wed Mar 20 21:58:20 2002
+++ bin86-0.16.12/ld/Makefile	Sun Sep  7 22:43:43 2003
@@ -17,7 +17,7 @@
 
 # An alternative file for a non-standard a.out.h (eg i386 linux on an Alpha)
 #
-# NATIVE=-DA_OUT_INCL='"a_out_local.h"' 
+NATIVE=-DA_OUT_INCL='"x86_aout.h"' 
 
 OBJS= dumps.o io.o ld.o readobj.o table.o typeconv.o linksyms.o \
       writex86.o writebin.o writerel.o
